packages/react-native-web/src/exports/AppRegistry/__tests__/__snapshots__/index-test.js.snap

```diff
@@ -44,7 +44,7 @@ input::-webkit-inner-spin-button,input::-webkit-outer-spin-button,input::-webkit
 .rn-paddingLeft-gy4na3{padding-left:0px}
 .rn-position-bnwqim{position:relative}
 .rn-zIndex-1lgpqti{z-index:0}
-.rn-flexGrow-16y2uox{-ms-flex-positive:1;-webkit-flex-grow:1;flex-grow:1}
+.rn-flexGrow-16y2uox{-ms-flex-positive:1;-webkit-box-flex:1;-webkit-flex-grow:1;flex-grow:1}
```

Comment:
I've no idea why this snapshot changed. I guess that there's some dependency to generate the prefixes and the minor upgrade added a new alias?

Reply:
Never-mind! This change seems to be very much in your interest 😊  https://github.com/rofrischmann/inline-style-prefixer/issues/153